tests/test-generator/src/processors.rs

```diff
@@ -0,0 +1,101 @@
+//! Here we define the processors usable for each test genrator
+use crate::{extract_name, Test, Testsuite};
+use std::path::PathBuf;
+
+/// Given a Testsuite and a path, process the path in case is a wast
+/// file.
+pub fn wast_processor(out: &mut Testsuite, p: PathBuf) -> Option<Test> {
+    let ext = p.extension()?;
+    // Only look at wast files.
+    if ext != "wast" {
+        return None;
+    }
+
+    // Ignore files starting with `.`, which could be editor temporary files
```

Comment:
There are crates that do file globbing that respect `.gitignores` I think 🤔 , might be simpler

Reply:
Just looked into it. The most obvious crate is:
https://crates.io/crates/ignore

But I think it's not trivial (as a few mins of work) to add it. If required we can add it in a PR apart. The simple solution works for now though :)